tendrl/gluster_integration/sds_sync/cluster_status.py

```diff
@@ -54,12 +54,14 @@ def sync_cluster_status(volumes, sync_ttl):
         )
 
     # Persist the cluster status
-    NS.gluster.objects.GlobalDetails(
+    gd_obj = NS.gluster.objects.GlobalDetails(
         status='healthy' if is_healthy else 'unhealthy',
         peer_count=peer_count,
         vol_count=len(volumes),
         volume_up_degraded=degraded_count
-    ).save(ttl=sync_ttl)
+    )
+    gd_obj.invalidate_hash()
+    gd_obj.save(update=False, ttl=sync_ttl)
```

Comment:
I would say you simply call self.invalidate_hash in the save() for GlobalDetails,  not here